slack_sdk/web/slack_response.py

```diff
@@ -1,11 +1,13 @@
 """A Python module for interacting and consuming responses from Slack."""
 
 import logging
-from typing import Union
+from typing import TypeVar, Union
```

Comment:
Can you add Optional to the typing imports?
```suggestion
from typing import TypeVar, Union, Optional
```

Reply:
Oof! That's what I get for going too fast.

Sorry. 😞